lib/chef/server_api.rb

```diff
@@ -31,6 +31,7 @@ def initialize(url = Chef::Config[:chef_server_url], options = {})
       options[:client_name] ||= Chef::Config[:node_name]
       options[:signing_key_filename] ||= Chef::Config[:client_key]
       options[:signing_key_filename] = nil if chef_zero_uri?(url)
+      options[:inflate_json_class] = false
```

Comment:
nah, it just flips from `JSONCompat.from_json` to `JSONCompat.parse` (which is just a wrapper over `FFI_Yajl::Parser.parse(source, opts)` )